docs/ENVVARS.md

```diff
@@ -2,36 +2,34 @@
+| `POSTGRES_DATABASE=tinkerbell`                    | string | server     | name of the PostgreSQL database for use in the Tink server                                                                                    |
+| `POSTGRES_PASSWORD=tink`                          | string | server     | PostgreSQL password for connecting to the DB                                                                                                  |
+| `POSTGRES_SSLMODE=disable`                        | string | server     | sets the PostgreSQL SSL priority [docs](https://www.postgresql.org/docs/10/libpq-connect.html#LIBPQ-CONNECT-SSLMODE)                          |
+| `POSTGRES_USER=tink`                              | string | server     | PostgreSQL username for connecting to the DB                                                                                                  |
+| `REGISTRY_PASSWORD=`                              | string | worker     | the password for the docker registry                                                                                                          |
+| `REGISTRY_USERNAME=`                              | string | worker     | the username for the docker registry                                                                                                          |
+| `RETRY_INTERVAL=`                                 | int    | worker     | the interval in seconds between retries for setting up connections to, querying for workflows from, and sending status reports to Tink Server |
+| `TINK_CLI_VERSION="0.0.0"`                        | string | cli        | if set to `0.0.0`, the old get command is used                                                                                                |
+| `TINKERBELL_CERTS_DIR=/certs`                     | string | server     | same as `CERTS_DIR`, deprecated in server                                                                                                     |
+| `TINKERBELL_CERT_URL=http://127.0.0.1:42114/cert` | string | clients    | url from which to get a TLS certificate, needed when Tink Server's TLS cert is signed by an unknown certificate authority, ie self-signed     |
+| `TINKERBELL_GRPC_AUTHORITY=127.0.0.1:42113`       | string | all        | same as `GRPC_AUTHORITY`, deprecated in server                                                                                                |
+| `TINKERBELL_HTTP_AUTHORITY=127.0.0.1:42114`       | string | server     | same as `HTTP_AUTHORITY`, deprecated in server                                                                                                |
+| `TINKERBELL_TLS="true"`                           | string | all        | configures grpc service or client connections for tls vs plaintext                                                                            |
```

Comment:
Did you generate this by hand?

Reply:
@jacobweinstock first committed this, by hand indeed I think.